### DISCUSSION TITLE
I have checked jenkins , gitea, harbor , but argocd and kubernetes aren't

### DISCUSSION THREAD

**Discussion by HoangGuruu** [ecoricemon/kube#4] [Q&A]
![image](https://github.com/ecoricemon/auto-kube/assets/111829092/e6745e4d-54e7-4847-9ccf-dffd28bb0f34)

**Comment by HoangGuruu** [comment]
Please tell me what i need to do , maybe step by step because it's confused with me , please

**Reply by ecoricemon** [reply]
Hello HoangGuruu!

In my opinion, your "vmext" isn't configured appropriately.

Your Jenkins, Gitea, and Harbor are working correctly.
So your "vmdesk" must have been able to reach "vmext", right?
But, it doesn't mean that "vmdesk" can reach Kubernetes cluster(vm0 and vm1).

Why don't you check out your "vmdesk" is capable of accessing to "vm0"?

Run following commands on "vmdesk"
$ ping 10.0.3.10
I guess it would fail. Then check this out.

Run following commands on "vmext"
$ sudo iptables-save | grep enp0s8
Can you see some rules you made using iptables command in the tutorial?
If you couldn't see them, please re-do this tutorial section
https://github.com/ecoricemon/auto-kube/blob/main/docs/tutorial/tutorial_vbox.md#configure-more-settings-for-connection-through-vmext
